_data/changelogs/component-identifier.yml

```diff
@@ -2,6 +2,14 @@ title: Identifier
 type: component
 changelogURL:
 items:
+  - date: 2024-05-31
+    summary: Fixed a bug that mistakenly added the word "An" to Spanish variants in the component preview.
+    summaryAdditional:
+    affectsGuidance: true
+    affectsContent: true
```

Comment:
```suggestion
    affectsContent: true
    affectsMarkup: true
```

Reply:
Lets reserve the markup key only for changes that affect the structure of the component